Source/Dafny/Dafny.atg

```diff
@@ -4486,6 +4529,24 @@ Expressions<.List<Expression> args.>
   }
   .
 
+/*------------------------------------------------------------------------*/
+TupleArgs<.List<ActualBinding> args.>
+= (. ActualBinding binding; bool isGhost = false; .)
+  // The IF is to distinguish between `(ghost var x := 5; x + x)` and (ghost x), both of which begin with `( ghost`.
+  // TODO(fpoli): It seems the linear-types branch also allows ghost match constructs, which is why they also need
+  // !IsMatch(). Should we do the same?
```

Comment:
Actually, I'm not sure what a ghost match is. Do you know any example that might need a `ghost` in correspondence with a `match`? If not, I'll just remove the TODO.

Reply:
We shouldn't need it. Remove.